webapp/src/js/services/markdown.js

```diff
@@ -17,7 +17,7 @@ angular.module('inboxServices').factory('Markdown',
       html = html.replace(/^##### (.*)\n/gm, '<h5>$1</h5>');
       html = html.replace(/__([^\s]([^_]*[^\s])?)__/gm, '<strong>$1</strong>');
       html = html.replace(/\*\*([^\s]([^*]*[^\s])?)\*\*/gm, '<strong>$1</strong>');
-      html = html.replace(/_([^_\s]([^_]*[^_\s])?)_/gm, '<em>$1</em>');
+      html = html.replace(/\s_([^_\s]([^_]*[^_\s])?)_/gm, ' <em>$1</em>');
```

Comment:
I don't think we currently have unit tests for this service, which is not great. Can you please start a new unit test? It doesn't need to test everything but lets make sure the basics work, and we can extend it as we go.

Either that or find a decent 3rd party library to format it for us...

Reply:
This service has been deleted.